sale_invoice_policy/models/product_template.py

```diff
@@ -28,6 +27,14 @@ def _default_default_invoice_policy(self):
         inverse="_inverse_invoice_policy",
     )
 
+    @api.model
+    def default_get(self, fields_list):
```

Comment:
@kimirondon Why does the default lambda not work ?

Reply:
Hello @rousseldenis, sorry for my lateness!, about the default, i had an issue with the method _reflect_selections,
but i believe #1762  will be merge soon.

Reply:
@kimirondon If you want to review that one and close this.